src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -1,71 +1,78 @@
+import { BaseSyntheticEvent } from 'react';
+import { stardust } from '@nebula.js/stardust';
 import * as handleAccessibility from '../handle-accessibility';
 
 describe('handle-accessibility', () => {
-  let cell;
-  let rootElement;
-  let focusedCellCoord;
-  let setFocusedCellCoord;
-  let keyboard;
+  let focusType: string;
+  let cell: HTMLTableCellElement | undefined;
```

Comment:
I do not think that undefined is needed here.

Reply:
That is the type that the function wants. And we do a sanity check in there, early return on !cell. So I think it can be undefined

Reply:
Yes, Max explanation is true, there is a sanity check for that.

Reply:
yeah, good